crates/qt-hello-world/build.rs

```diff
@@ -0,0 +1,17 @@
+use cxx_qt_build::{CxxQtBuilder, QmlModule};
+
+fn main() {
+    CxxQtBuilder::new()
+        // - Qt Core is always linked
+        // - Qt Qml is linked by enabling the qt_qml Cargo feature.
+        // - Qt Qml requires linking Qt Network on macOS
+        .qt_module("Network")
+        .qt_module("Quick")
+        .qml_module(QmlModule::<&str, _> {
```

Comment:
why do we need the `::<&str, _>`

Reply:
The list of Rust files is empty in this example, so we need to tell the QMLModule template what type the entries of that list are.
In this case it doesn't really matter, but we do not to specify something, as type inference can't find a type to go off of.